### PR TITLE
Feat: Reset tutorials

### PR DESCRIPTION
**Pull request by bhoopesh369**

### CODE REVIEW

File: src/components/BattleTV/BattleTV.module.css [hunk 1]
```diff
@@ -261,3 +261,21 @@
...
+  font-family: monospace, sans-serif;
+  display: inline-block;
+  white-space: nowrap;
+  transition: 0.3s;
+}
+
+.button:hover {
+  color: #0a1c2a;
+  background-color: #aaaaaa;
+}
```
Comment (Ram-20062003) [inline src/components/BattleTV/BattleTV.module.css]: add EOF

File: src/components/DashboardOptions/DashboardOptions.tsx [hunk 2]
```diff
@@ -1,15 +1,44 @@
...
 
 interface dashboardoptions {
   image?: JSX.Element;
   onLogout?: React.MouseEventHandler<HTMLElement>;
 }
 
 const DashboardOptions = (props: dashboardoptions): JSX.Element => {
+  const currentUserApi = new CurrentUserApi(apiConfig);
+  const navigate = useNavigate();
+  const User = useAppSelector(user);
```
Comment (Ram-20062003) [inline src/components/DashboardOptions/DashboardOptions.tsx]: why getting a user here? not needed.

File: src/components/DashboardOptions/DashboardOptions.tsx [hunk 3]
```diff
@@ -1,15 +1,44 @@
...
+
+  const dispatch = useAppDispatch();
+
+  const resetTutorials = () => {
+    currentUserApi
+      .updateCurrentUser({
+        name: User.name,
+        country: User.country,
+        college: User.college,
```
Comment (Ram-20062003) [inline src/components/DashboardOptions/DashboardOptions.tsx]: Remove redundant updates

File: src/components/DashboardOptions/DashboardOptions.tsx [hunk 4]
```diff
@@ -1,15 +1,44 @@
...
+        updateTutorialLevel: 'RESET',
+      })
+      .then(() => {
+        navigate('/dashboard');
+        dispatch(isTourResetChanged(true));
```
Comment (Ram-20062003) [inline src/components/DashboardOptions/DashboardOptions.tsx:32]: Also make isTourOver false here to open that model
```suggestion
        dispatch(isTourOverChanged(false));
        dispatch(isTourResetChanged(true));
```

File: src/components/Leaderboard/Leaderboard.tsx [hunk 5]
```diff
@@ -160,7 +160,10 @@ function PaginatedItems() {
                               className={styles.pic}
                               src={getAvatarByID(row.user.avatarId).url}
                             ></img>
-                            {' ' + row.user.username}
+                            {' ' +
+                              (row.user.username.length > 100
+                                ? row.user.username.substring(0, 10) + '...'
+                                : row.user.username)}
```
Comment (Ram-20062003) [inline src/components/Leaderboard/Leaderboard.tsx]: - Always have first 10 char 
```suggestion
                                row.user.username.substring(0, 10)
```
- Do the similar update in `DC-LeaderBoard` and `BattleTv`

File: packages/client/tsconfig.esm.json [hunk 6]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (Ram-20062003) [inline packages/client/tsconfig.esm.json]: remove pls

**Review by Ram-20062003 — CHANGES_REQUESTED** [review]

**Review by Ram-20062003 — APPROVED** [review]

**Review by Ram-20062003 — APPROVED** [review]

**Review by vigneshd332 — APPROVED** [review]

**Pull request merged**